update-readme (#24)
RoadieHQ/backstage-roadie-plugins: plugins/backstage-plugin-github-pull-requests/README.md

![a list of pull requests in the GitHub Pull Requests](https://raw.githubusercontent.com/RoadieHQ/backstage-plugin-github-pull-requests/main/docs/list-of-pull-requests-and-stats-tab-view.png)

> ## Notice

Reviewer: ```suggestion
## Repository migration notice
```

I want to make this heading a little more alarming so people read it.

> If you have previously used this plugin you will be noticing some changes. In order to increase usability of our plugins and make testing and integration process straightforward and easy we are currently working on re-structuring and re-organising our plugins workflows and architecture.

Reviewer: My suggested version basically says the same thing, just in a shorter fashion. It's also important to tell the users what is not changing, to proactively answer the top questions that may pop up in their minds.

```suggestion
In order to make testing and deployment of our plugins easier we are migrating all Roadie plugins to a monorepo at https://github.com/RoadieHQ/backstage-roadie-plugins.

The plugins will still be published to the same place on NPM and will have the same package names so nothing should change for consumers of these plugins.
```

> We have decided to gather all of our plugins repositories in one place, so it is easier to find everything you need, which will boost the usage and hopefully give better overview of everything you can achieve with Backstage.

Reviewer: ```suggestion
```

> From now on, this plugin will be contained inside https://github.com/RoadieHQ/backstage-roadie-plugins repository, which will gather all of the plugins we, at Roadie, have worked on.

Reviewer: ```suggestion
```

> Updated instructions could be found below:

Reviewer: ```suggestion
```
